examples/02-plot/distance-along-spline.py

```diff
@@ -0,0 +1,75 @@
+"""
+.. _distance_labels_example:
+
+Label based on Distance on Line
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Create a spline and generate labels along the spline based on distance along a
+spline.
+
+This is an extension of the :ref:`create_spline_example`.
+
+"""
+
+import numpy as np
+
+import pyvista as pv
+
+###############################################################################
+# Create a spline
+# ~~~~~~~~~~~~~~~
+# Create a spline using :func:`pyvista.Spline`.
+
+# Make points along a spline
+theta = np.linspace(-1 * np.pi, 1 * np.pi, 100)
+z = np.linspace(2, -2, 100)
+r = z**2 + 1
+x = r * np.sin(theta)
+y = r * np.cos(theta)
+points = np.column_stack((x, y, z))
+
+# Create a spline. This automatically computes arc_length, which is the
+# distance along the line.
+spline = pv.Spline(points, 1000)
+spline.point_data
+
+
+###############################################################################
+# Determine the coordinates matching distance along a spline
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Here we write a simple function that gets the closest point matching a distance along a spline and then generate labels for those points.
+
+
+def get_point_along_spline(distance):
+    """Return the closest point on the spline a given a length along a spline."""
```

Comment:
```suggestion
    """Return the closest point on the spline a given length along a spline."""
```

Reply:
Good catch, but I think this is supposed to be the other way around:
```suggestion
    """Return the closest point on the spline given a length along the spline."""
```